src/components/ebay-infotip/template.marko

```diff
@@ -0,0 +1,32 @@
+<ebay-tooltip-base
+    type="infotip"
+    location=data.location
+    style-left=data.styleLeft
+    style-top=data.styleTop
+    style-right=data.styleRight
+    style-bottom=data.styleBottom>
+    <span
+        class="infotip"
+        ${data.htmlAttributes}
+        w-on-expander-expand='handleExpand'
+        w-on-expander-collapse='handleCollapse'
+        w-bind>
+        <button
+            class='infotip__host icon-btn'
+            aria-describedby=`infotip-${widget.id}`
+            aria-label=data.ariaLabel>
+            <div if(data.icon) body-only-if(true)>
```

Comment:
Here we should replace the `<div if(data.icon) body-only-if(true)>` with `<if(data.icon)>` as they achieve the same output.